main.rb

```diff
@@ -0,0 +1,123 @@
+require_relative 'planet'
+require_relative 'solar_system'
+require 'pry'
+
+# Welcome message to user
+def welcome_message
+  puts "=" * 80
+  puts "\t\t\tHello and Welcome to THE MAGIC SCHOOL BUS!"
+  puts "\tOn today's adventure, we will be exploring the Solar System of Anirtak..."
+  puts "=" * 80
+  puts "Let's choose from our options below! "
+  puts "|| A || : Explore a list of all the planets"
+  puts "|| B || : Discover all the details about a planet"
+  puts "|| C || : Add your own planet"
+  puts "|| D || : Find the distance between two planets"
+  puts "|| Q || : Hop off the bus, and exit!"
+
+  main
+end
+
+# Main method that interacts with user
+def main
+  xandar = Planet.new("XANDAR","aquamarine powder blue", 6.39e23, 2.279e8,
+    "It's been known to host living creatures that are closely related to smurfs")
+  onurb = Planet.new("ONURB", "shades of lemon, light salmon, and honeydew",
+  1.898e27, 7.785e8, "It does not have a true surface, as the planet is mostly swirling gases and liquids")
+
+  solar_system = SolarSystem.new("Anirtak")
+  solar_system.add_planet(xandar)
+  solar_system.add_planet(onurb)
+
+  # loop until user quits
+  user_play = "Y"
+  while user_play == "Y"
+    print  "Enter A, B, C, D or Q: "
+    user_input = gets.chomp.strip.upcase
+    while user_input !='A' &&
+      user_input != 'B' &&
+      user_input != 'C' &&
+      user_input != 'D' &&
+      user_input != 'Q'
+      print "EEP. Please try again: "
+      user_input = gets.chomp.strip.upcase
+    end
+    list = solar_system.list_planets
+
+      if user_input == 'A'
```

Comment:
Instead of checking what `user_input` is equal to with a bunch of `if`/`elsif`/`else` clauses, what else could we do?